### PR TITLE
forward porting linux packaging updates to fix native pkgs

### PR DESCRIPTION
**Pull request by kensipe**
JIRA issues:   MARATHON-8522

### CODE REVIEW

File: ci/pipeline [hunk 1]
```diff
@@ -368,5 +368,5 @@ def release(args: String*): Unit = {
       %('sbt, "plugin-interface/publish")
   }
 
-  githubClient.tag(config.version.commit, config.version.toTagString)
+  // githubClient.tag(config.version.commit, config.version.toTagString)
```
Comment (timcharper) [inline ci/pipeline]: can we just delete this line? I never want the release job doing tags. It's too weird.

Comment (jeschkies) [reply ci/pipeline]: What is the motivation for removing this line?

File: ci/releases.sc [hunk 2]
```diff
@@ -102,24 +102,28 @@ def uploadLinuxPackagesToRepos(tagName: String): Unit = {
   // Note - the key is expected to be provided via an SSH agent
   utils.printStageTitle(s"Uploading native packages")
   %("rsync", "-avz",
-    (pwd / 'target / 'packages) + "/",
-    s"${pkgserverUser}@${pkgserverHost}:repo/incoming/marathon-${tagName}/")
+    (pwd / 'tools / 'packager) + "/",
+    s"${pkgserverUser}@${pkgserverHost}:repo/incoming/marathon-${tagName}/",
+    "--include", "*.deb",
+    "--include", "*.rpm",
+    "--exclude", "*")
```
Comment (jeschkies) [inline ci/releases.sc]: Did we upload too much?

File: ci/releases.sc [hunk 3]
```diff
@@ -102,24 +102,28 @@ def uploadLinuxPackagesToRepos(tagName: String): Unit = {
...
-    "upstart" -> s"ubuntu/trusty${pkgType}",
-    "systemv" -> s"el${pkgType}/6",
-    "systemd" -> s"el${pkgType}/7")
+    "debian8" -> s"debian/jessie${pkgType}",
+    "debian9" -> s"debian/stretch${pkgType}",
+    "ubuntu1604" -> s"ubuntu/xenial${pkgType}",
+    "ubuntu1404" -> s"ubuntu/trusty${pkgType}",
+    "ubuntu1804" -> s"ubuntu/bionic${pkgType}",
+    "el6" -> s"el${pkgType}/6",
+    "el7" -> s"el${pkgType}/7")
```
Comment (jeschkies) [inline ci/releases.sc]: 🙈 oh my. I broke that, didn't I?

File: ci/pipeline (hunk 1 above)
Comment (kensipe) [reply ci/pipeline]: 1. the CI job doesn't have authorization to tag
2. there is less of a reason to tag something that has a nature tag (the rev is in the build)
3. we have traditional manually affix a release tag which will require human involvement to provide release not details.

Comment (jeschkies) [reply ci/pipeline]: Ah, ok, I thought each community release has to have a tag. Feel free to remove this line.

**Pull request merged**